src/main.rs

```diff
@@ -21,12 +21,14 @@ use tracing_subscriber::fmt::writer::MakeWriterExt;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
-	let xdg = xdg::BaseDirectories::with_prefix("stratawm")?;
+	let xdg = xdg::BaseDirectories::with_prefix("strata")?;
 
 	let config_dir = xdg.find_config_file("");
 	let lib_dir = xdg.find_data_file("lua");
 	let log_dir = xdg.get_state_home();
 
+	println!("{:?}", lib_dir);
```

Comment:
This is probably a leftover from your tests :sweat_smile:

Reply:
Oh yeah, forgot to remove that :sweat_smile: